src/extension.ts

```diff
@@ -950,6 +979,13 @@ export function beautify(): void {
                 } else {
                     newLine = newLine.trimEnd();
                 }
+                if (isVariableDeclaration && !newLine.startsWith("#VAR"))
+                {
+                    let spaces = longestVariableDeclarationLine - newLine.indexOf("=");
+                    let left = newLine.substring(0, newLine.indexOf("="));
+                    let right = newLine.substring(newLine.indexOf("="));
+                    newLine = " ".repeat(whiteSpaces) + left + " ".repeat(spaces) + right;
+                }
                 outputChannel.appendLine(newLine);
                 textEdits.push(vscode.TextEdit.replace(line.range, newLine));
             }
```

Comment:
I would consider a small improvement for the following results:
e.g:
Chaotic
```
#VAR
  V.L.SurfacePosition            = 0                       
  V.L.RetractionPlane =                       50                     
  V.L.SafetyClearance       = 2                      
#ENDVAR
```
Beautified right now:
```
#VAR
  V.L.SurfacePosition            = 0
  V.L.RetractionPlane            =                       50
  V.L.SafetyClearance            = 2
#ENDVAR
```
Beautified with possible improvement:
```
#VAR
  V.L.SurfacePosition = 0
  V.L.RetractionPlane = 50
  V.L.SafetyClearance = 2
#ENDVAR
```
I guess therfore we would need to do the left-right-seperation earlier to trim arround the equal sign and then do the used behaviour.
What do you think of this?

Reply:
I think you are right. Thank you for taking the time to look at the code and your suggestions for improvement.